pkg/test/resourcefixture/testdata/basic/compute/v1beta1/computemanagedsslcertificate/managedsslcertificate/create.yaml

```diff
@@ -1,4 +1,4 @@
-# Copyright 2023 Google LLC
+# Copyright 2024 Google LLC
```

Comment:
Have you got the chance to run the dynamic test locally? There is no update.yaml so I think you'll want to add this test case into compute_context.go and explain why update is not supported.